app/models/metric/capture.rb

```diff
@@ -128,32 +127,35 @@ def self.perf_capture_health_check(zone)
   end
   private_class_method :perf_capture_health_check
 
+  # Collect realtime targets and group them by their rollup parent
+  #
+  # 1. Only display rollups for Hosts
+  # 2. Since perf_target_to_interval_name(Host) == "realtime", no need to filtering on interval_name == "realtime"
+  # 3. Some Hosts have an EmsCluster as a parent, others have none.
+  #
+  # @returns Hash<String,Array<Host>>
+  #   e.g.: {"EmsCluster:4"=>[Host:4], "EmsCluster:5"=>[Host:1, Host:2]}
```

Comment:
Given point 3 above, maybe have that here in the example as to what that looks like?